js/index.js

```diff
@@ -34,7 +33,6 @@ window.onload = () => {
     request.onload = () => {
```

Comment:
Rather than repeat this entire code. It's best you abstract it into a function and call the function when the browser loads and when a user clicks search.

Reply:
Will do that now. Thank you

Reply:
Done